packages/api/cms-api/src/generator/generate-crud-input.ts

```diff
@@ -372,7 +372,7 @@ export async function generateCrudInput(
 import { Transform, Type } from "class-transformer";
 import { IsString, IsNotEmpty, ValidateNested, IsNumber, IsBoolean, IsDate, IsOptional, IsEnum, IsUUID, IsArray, IsInt } from "class-validator";
 import { IsSlug, RootBlockInputScalar, IsNullable, PartialType} from "@comet/cms-api";
-import { GraphQLJSONObject } from "graphql-type-json";
+import { GraphQLJSONObject } from "graphql-scalars";
```

Comment:
I'm sorry, but I believe this is a breaking change. If a developer currently uses graphql-type-json and re-runs the API Generator after a minor Comet update, a change is necessary to fix the following error:

<img width="947" alt="image" src="https://github.com/vivid-planet/comet/assets/48853629/ed82c263-8313-42c1-8535-d64a2b0bd76e">

@nsams could we solve this by choosing the right import based on the `@Field(() => GraphQLJSONObject)` annotation?

Reply:
We can also simply decide to make this breaking and target next along with #2192

Reply:
Yes, let's target next instead (stable v7 should be only a few days/weeks away anyway)